erigon-lib/common/disk/common.go

```diff
@@ -0,0 +1,25 @@
+package disk
+
+import (
+	"context"
+	"time"
+
+	"github.com/ledgerwatch/log/v3"
+)
+
+func UpdateDiskStats(ctx context.Context, logger log.Logger) {
+	logEvery := time.NewTicker(180 * time.Second)
```

Comment:
@AskAlexSharov I was wondering if 180 seconds is too long for this metric, should we shorten it? If yes, what would be suitable?

Reply:
fine for me